packages/linux-firmware/meson-firmware/package.mk

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="meson-firmware"
+PKG_VERSION="edd24b481293b93814494508cd4952b67f15acb3"
+PKG_SHA256="2092c71a5eb106725784dadcc2cfea7be60254539a41fafebf311235283259e4"
+PKG_LICENSE="GPLv2"
+PKG_SITE="https://github.com/LibreELEC/meson-firmware"
+PKG_URL="https://github.com/LibreELEC/meson-firmware/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_LONGDESC="meson-firmware: Amlogic microcode firmware for the V4L2 mem2mem vdec driver"
+PKG_TOOLCHAIN="manual"
+
+makeinstall_target() {
+  FW_TARGET_DIR=$INSTALL/$(get_full_firmware_dir)
+
+  if find_file_path firmwares/$PKG_NAME.dat; then
```

Comment:
After https://github.com/LibreELEC/LibreELEC.tv/pull/3517/commits/3567efcbe81c5fef8dc3df477fccbe64e3478862 can you change `firmwares` to `config`